plugins/metrics/runtime/runtime.go

```diff
@@ -0,0 +1,286 @@
+package runtime // import "go.opentelemetry.io/contrib/plugins/metrics/runtime"
```

Comment:
A `doc.go` comment file would be helpful listing all the metrics produced here. It's fine to make it a TODO, since we haven't actually created a conventional standard which it should be able to link to. How about:

> package runtime implements the work-in-progress conventional runtime metrics specified by OpenTelemetry. The metrics produced are:
> [list of metrics and brief descriptions]

Reply:
Will do.